### PR TITLE
Automatic filters from json add tests

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

File: ITKImageProcessingFilters/ITKBilateralImage.cpp [hunk 1]
```diff
@@ -153,3 +153,10 @@ AbstractFilter::Pointer ITKBilateralImage::newFilterInstance(bool copyFilterPara
 const QString ITKBilateralImage::getHumanLabel()
 { return "[ITK] ITK  Bilateral Image Filter (KW)"; }
 
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+const QString ITKBilateralImage::getSubGroupName()
+{ return "ImageEnhancement"; }
```
Comment (thewtex) [inline ITKImageProcessingFilters/ITKBilateralImage.cpp]: ITKImageEnhancement?

File: ITKImageProcessingFilters/ITKIntensityWindowingImage.cpp [hunk 2]
```diff
@@ -156,3 +156,10 @@ AbstractFilter::Pointer ITKIntensityWindowingImage::newFilterInstance(bool copyF
 const QString ITKIntensityWindowingImage::getHumanLabel()
 { return "[ITK] ITK  Intensity Windowing Image Filter (KW)"; }
 
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+const QString ITKIntensityWindowingImage::getSubGroupName()
+{ return "IntensityImageFilters  MultiThreaded"; }
```
Comment (thewtex) [inline ITKImageProcessingFilters/ITKIntensityWindowingImage.cpp]: ITKImageIntensity?

File: Utilities/SimpleITKJSONDream3DFilterCreation.py [hunk 3]
```diff
@@ -665,6 +666,18 @@ def GetDREAM3DParameterDescription(filter_member):
...
+def FindModuleInIncludeFile(name, path):
+    for root, dirs, files in os.walk(path):
+        if name in files:
+            file_path = os.path.join(root, name)
+            datafile = file(file_path)
+            for line in datafile:
+                if '\ingroup' in line:
+                    split_line=line.split('\ingroup')
+                    return split_line[1].strip()
+    return "ITKNoModule"
```
Comment (thewtex) [inline Utilities/SimpleITKJSONDream3DFilterCreation.py:706]: May also need a check for starting with "ITK".

Comment (fbudin69500) [reply Utilities/SimpleITKJSONDream3DFilterCreation.py:706]: Thanks for the review!

**Review by thewtex — APPROVED** [review]

**Pull request merged**